lib/server/forwardClientRequestToApi.js

```diff
@@ -0,0 +1,39 @@
+"use strict";
+
+var request = require("request-promise");
+
+function forwardClientRequestToApi(apiHost){
+    return function(req,res){
+        var options = {
+            uri: apiHost + req.url.replace(/^\/api/,''),
+            method: req.method
+        };
+        request(options)
+        .then(function(response) {
+            res.send(response);
+        })
+        .catch(function(error) {
+            res.status(error.statusCode || 500)
+                .json({
+                    message: error.message
```

Comment:
if the api requests are forwarded, i don't think brisket should create a custom error message format. if we just do a pass through, it'll be up to the api and application to build their own error message contract